packages/cloud-cognitive/src/components/EditTearsheet/preview-components/MultiStepTearsheet.js

```diff
@@ -0,0 +1,298 @@
+/* eslint-disable react/prop-types */
+/**
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import {
+  Button,
+  Column,
+  InlineNotification,
+  RadioButtonGroup,
+  RadioButton,
+  Row,
+  TextInput,
+  Toggle,
+  NumberInput,
+} from 'carbon-components-react';
+import cx from 'classnames';
+import { pkg } from '../../../settings';
+import { EditTearsheet } from '../EditTearsheet';
+import { EditTearsheetStep } from '../EditTearsheetStep';
+
+const blockClass = `${pkg.prefix}--tearsheet-edit-multi-step`;
+
+export const MultiStepTearsheet = ({
+  backButtonText,
+  cancelButtonText,
+  className,
+  description,
+  influencerWidth,
+  label,
+  nextButtonText,
+  submitButtonText,
+  title,
+}) => {
+  const [simulatedDelay] = useState(750);
+  const [open, setOpen] = useState(false);
+  const [shouldReject, setShouldReject] = useState(false);
+  const [hasSubmitError, setHasSubmitError] = useState(false);
+  const [stepOneTextInputValue, setStepOneTextInputValue] =
+    useState('Topic name here');
+  const [topicDescriptionValue, setTopicDescriptionValue] = useState(
+    'Topic description here'
+  );
+  const [topicVersionValue, setTopicVersionValue] =
+    useState('Topic value here');
+  const [topicLocationValue, setTopicLocationValue] = useState('Location here');
+  const [stepTwoTextInputValue, setStepTwoTextInputValue] = useState(1);
+  const [stepThreeTextInputValue, setStepThreeTextInputValue] =
+    useState('one-day');
+  const [isInvalid, setIsInvalid] = useState(false);
+
+  const clearCreateData = () => {
+    setStepOneTextInputValue(stepOneTextInputValue);
+    setTopicDescriptionValue(topicDescriptionValue);
+    setTopicVersionValue(topicVersionValue);
+    setTopicLocationValue(topicLocationValue);
+    setStepTwoTextInputValue(1);
+    setStepThreeTextInputValue('one-day');
+    setHasSubmitError(false);
+    setIsInvalid(false);
+    setOpen(false);
+  };
+
+  const handleModalClick = () => {
+    setOpen(!open);
+  };
+
+  return (
+    <div>
+      <style>{`.${blockClass} { opacity: 0 }`};</style>
+      <Button onClick={handleModalClick}>
+        {open ? 'Close EditTearsheet' : 'Open EditTearsheet'}
+      </Button>
+      <EditTearsheet
+        influencerWidth={influencerWidth}
+        label={label}
+        className={cx(blockClass, className)}
+        submitButtonText={submitButtonText}
+        cancelButtonText={cancelButtonText}
+        backButtonText={backButtonText}
+        nextButtonText={nextButtonText}
+        description={description}
+        title={title}
+        open={open}
+        onHandleModalClick={handleModalClick}
+        onClose={clearCreateData}
+        onRequestSubmit={() =>
```

Comment:
similar comment i made about having function calls like this in the HTML. i think you should consider moving these out into their own handlers. `onRequestSubmit={submitHandler}`